src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocPackageCheck
**Severity**: ERROR
**File**: /home/bjerre/workspace/violation-comments-to-github-plugin/work/jobs/test/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/MyClass.java L0

Missing package-info.java file.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*